crossdock/server/endtoend.py

```diff
@@ -99,7 +101,7 @@ def generate_traces(self, request, response_writer):
         tracer = self.tracers[sampler_type]
         for _ in range(req.get('count', 0)):
             span = tracer.start_span(req['operation'])
-            for k, v in req.get('tags', {}).iteritems():
+            for k, v in req.get('tags', {}).items():
```

Comment:
Ok addressed this in https://github.com/uber/jaeger-client-python/pull/43/commits/2612c733056807cbf8f4c9272066e1861d564097